cloudbuild.yaml

```diff
@@ -287,10 +289,15 @@ steps:
         set -e
 
         cd /workspace/infrastructure
-        terraform destroy -var-file=tfvars_tests/standard-gke-public.platform.tfvars -var=project_id=$PROJECT_ID \
+        terraform destroy \
+        -var-file=platform.tfvars \
+        -var=project_id=$PROJECT_ID \
+        -var=network_name=ml-network-$_BUILD_ID \
+        -var=subnetwork_name=ml-network-$_BUILD_ID \
         -var=cluster_name=ml-$SHORT_SHA-$_PR_NUMBER-$_BUILD_ID-cluster \
         -var=autopilot_cluster=$_AUTOPILOT_CLUSTER \
-        -var=cluster_location=$_REGION -auto-approve -no-color
+        -var=cluster_location=$_REGION \
```

Comment:
Also consider changing the region to us-east4

Reply:
Done